index.bs

```diff
@@ -2435,12 +2464,18 @@ To <dfn>match [=filter values=] with negation</dfn> given a [=filter value=] |a|
 1. If |i| is not [=set/is empty|empty=], then return false.
 1. Return true.
 
-To <dfn>match an attribution source's filter data against a filter map</dfn> given an
-[=attribution source=] |source|, a [=filter map=] |filter|, and a [=boolean=]
-<dfn for="match an attribution source's filter data against a filter map"><var>isNegated</var></dfn>:
+To <dfn>match an attribution source's filter data against a filter config</dfn> given an
```

Comment:
Do we want to distinguish the existing filter-match failure from lookback failure in the verbose debug reports?

As written, both types will use the existing `trigger-no-matching-filter-data` / `trigger-event-no-matching-configurations` types.

Reply:
Clearer tracing debug signals are helpful.

Reply:
One issue with reporting different failure types is that both could occur at the same time. e.g. First filter is outside the lookback window and the second has no match. We could:

1. Keep reporting one type of failure that encompasses both types.
2. Report a different type of failure by using a heuristic (lookback first, first seen, etc) to pick the type of failure to report on when both are applicable.
3. Report multiple failure types in a single verbose debug report.

I think 2 would be a reasonable option to give visibility on lookback window failures without the complexity associated with supporting multiple failure types.

Reply:
I don't think we should introduce new failure types as it is still a `filter-data` & `no-matching-configuration` failure along with being a `lookback_window` failure or a `source_type` failure

Maybe we can provide additional info about the failure along with the type i.e. sub-type/error_info within the verbose report.

Reply:
Based on https://github.com/WICG/attribution-reporting-api/pull/914#discussion_r1275459945 and https://github.com/WICG/attribution-reporting-api/pull/914#discussion_r1275528357 I think it's fine to use the existing debug report types. In today's debug reports, we don't say which filter field(s) failed to match, so if lookback is being implemented as a special type of filter field, I think it's consistent to keep things as they are.